docs/CONTRIBUTING.md

```diff
@@ -65,3 +65,13 @@ Note: If you have pre-installed [Poe](https://poethepoet.natn.io/index.html)
 ## Versioning
 
 Versioning follows [Semantic Versioning](https://semver.org/). For new features, bump the minor version. For bug fixes, bump the patch version. For pre-releases, append `dev.N` to the version. For example, `0.1.0dev.1` is the first pre-release of the `0.1.0` version.
+
+## GitHub Action Workflows
+
+This repo uses a policy of SHA-pinning GitHub Actions, for hardened security.
+
+To pin your GitHub actions, you can use the [pinact](https://github.com/suzuki-shunsuke/pinact) tool:
+
+```bash
+[tk] - bash CLI example here
+```
```

Comment:
@wennergr - Can you provide the CLI command you used here (optionally more instructions or context) for our contributing docs and our collective future reference?